### DISCUSSION TITLE
CCF with OnPrem data

### DISCUSSION THREAD

**Discussion by arunkumarviswanathan** [cloud-carbon-footprint/cloud-carbon-footprint#1267] [Q&A]
Is it possible to view the CCF UI with the OnPrem data? If yes please share the steps to achieve it.
On starting the CCF without any config, it shows up only the 4 public cloud providers, I need to display the OnPrem provider also and show the data from the OnPrem CSV file.

I appreciate any help you can provide.
Arun

**Comment by 4upz** [comment]
Hi @arunkumarviswanathan ,

Unfortunately, our on-premise feature is still in early development and is not yet available in the CCF UI. As a workaround, you're welcome to expand the dashboard of your CCF instance to integrate your data, or use other visualization tools such as [Looker Studio](https://lookerstudio.google.com/overview) or [QuickSight](https://aws.amazon.com/pm/quicksight/?gclid=Cj0KCQiAmNeqBhD4ARIsADsYfTcvSA6v98PCvDt2hzsgMSKsSoF4gzCycCFzqH95gAhUsQIuLyMK89AaAgC-EALw_wcB&trk=45a4ffb0-ee00-486d-af60-63655115b5d8&sc_channel=ps&ef_id=Cj0KCQiAmNeqBhD4ARIsADsYfTcvSA6v98PCvDt2hzsgMSKsSoF4gzCycCFzqH95gAhUsQIuLyMK89AaAgC-EALw_wcB:G:s&s_kwcid=AL!4422!3!652240143325!e!!g!!quicksights!19878796219!153028446651) to import and visualize your csv data.

We're hoping that it's something that we can implement soon. We're open to contributions and feature suggestions around this if there is a strong need from the community
